### PR TITLE
Refactor - Rename Metaparameters to AttributeSets

### PR DESCRIPTION
**Pull request by RossMurr4y**
## Description

* Rename the `Metaparameter` composite object to `AttributeSet`

## Motivation and Context

Contributes to #1491 

## How Has This Been Tested?

Local schema generation.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Refactor (non-breaking change which improves the structure or operation of the implementation)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Followup Actions

- [x] None

## Checklist:

- [x] My change requires a change to the [documentation](https://github.com/hamlet-io/docs).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] None of the above.

### CODE REVIEW

File: providers/shared/attributesets/attributeset.ftl [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[#ftl]
+
+[#assign LINK_ATTRIBUTESET_TYPE = "Link"]
```
Comment (ml019) [inline providers/shared/attributesets/attributeset.ftl:3]: Maybe make lower case in line with things like component types

**Review by ml019 — CHANGES_REQUESTED** [review]
AttributeSet is a better name - says what it is.

File: providers/shared/attributesets/attributeset.ftl (hunk 1 above)
Comment (RossMurr4y) [reply providers/shared/attributesets/attributeset.ftl]: @ml019 updated in https://github.com/hamlet-io/engine/pull/1498/commits/835a9271a905ef6794e7aec78907ef492cf2f157

**Comment by RossMurr4y** [comment]
@ml019 for your re-review when you have time.

**Review by roleyfoley — APPROVED** [review]

**Pull request merged**